### PR TITLE
BETA: Register nx.is-a.dev

### PR DESCRIPTION
**Pull request by ghost**
Added `nx.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/nx.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "owner": {
+        "username": "jadfoq",
+        "email": "jadfoq@getgoogleoff.me"
+    },
+    "record": {
+    "MX": [
```
Comment (DEV-DIBSTER) [inline domains/nx.json:7]: Invalid MX records. No TXT records provided? Also these appear to be NS records, could you clarify that these are MX addresses?

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/nx.json (hunk 1 above)
Comment (jadfoq) [reply domains/nx.json:7]: https://github.com/is-a-dev/register/blob/9e02fa2a5590e83d25cd91268f4a6b2d4b81797e/domains/nx.json
https://manage.is-a.dev bug

**Comment by jadfoq** [comment]
Can you use [this file](https://git.disroot.org/jadfoq/register/src/commit/3064471e76feacec904d6afa3cb19cd425ce0425/domains/nx.json), github doesn't work correctly?

**Comment by jadfoq** [comment]
I'm on the [ActivityPub](https://en.wikipedia.org/wiki/ActivityPub) network - https://fe.disroot.org/@jadfoq

**Comment by CuteDog5695** [comment]
> I'm on the [ActivityPub](https://en.wikipedia.org/wiki/ActivityPub) network - https://fe.disroot.org/@jadfoq

we dont support nameservers (also thats not the way to use nameservers, CNAME different to the one you want)

**Comment by jadfoq** [comment]
https://git.disroot.org/jadfoq/register/src/branch/main/domains/nx.json fix, please pull it

File: domains/nx.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jadfoq",
+        "email": "jadfoq@getgoogleoff.me"
+    },
+    "record": {
+    "A": "185.31.40.24"
```
Comment (CuteDog5695) [inline domains/nx.json:7]: A record needs to be in an array + site is not found

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/nx.json (hunk 2 above)
Comment (jadfoq) [reply domains/nx.json:7]: it is a alwaysdata ip https://help.alwaysdata.com/en/sites/use-external-addresses/

**Pull request closed without merging**